modules/optimization/test/tests/executioners/constrained/shape_optimization/forward.i

```diff
@@ -0,0 +1,278 @@
+inner_radius = 6
```

Comment:
Add a comment to the top of this file that it is documented as an example and should not be changed without updating the documentation.